playbooks/ReadMe.txt

```diff
@@ -0,0 +1,9 @@
+Automate provisioning OpenStack Instance
+
+- The Ansible playbook will manually create the OpenStack instance without having to open a virtual machine. 
+- The playbook will automate the OpenStack instance once it is run on the terminal.
+- The playbook contains the neccessary credentials to create the instance. 
+- The credentials are: image, key_name, flavor, network, and security_groups
```

Comment:
@LikhithaEda I think this is not correct. 
The credentials are something secure to authenticate. The ones you have mentioned are parameters to the module.

Reply:
ok.